extensions/wikia/ExactTargetUpdates/maintenance/ExactTargetUpdateUserEditsPerWikiMaintenance.php

```diff
@@ -75,6 +85,43 @@ private function getLastDayDate() {
 		$sStartDate = $oNow->format('Y-m-d H:i:s');
 		return $sStartDate;
 	}
+
+	/**
+	 * @param array $aUsersEditsDataChunk [ user_id@wiki_id => editcount ]
+	 * e.g.
+	 * [
+	 *  '12345@177' => 3,
+	 *  '15432@177' => 2
+	 * ]
+	 * Means user 12345 edited 3 times on 177 wikia and user 15432 edited 2 times on 177 wikia
+	 * @return array
+	 * e.g.
+	 * [
+	 *  '12345' => [ '177' => 3 ],
+	 *  '15432' => [ '177' => 2 ]
+	 * ]
+	 */
+	private function prepareDataFormat( $aUsersEditsDataChunk ) {
+		$aUsersEditsData = [];
+		foreach ( $aUsersEditsDataChunk as $sMergedKey => $iEditcount ) {
+			$aUnmergedKeysAssoc = $this->unmergeKey( $sMergedKey );
+			$iUserId = $aUnmergedKeysAssoc['user_id'];
+			$iWikiId = $aUnmergedKeysAssoc['wiki_id'];
+			$aUsersEditsData[$iUserId][$iWikiId] = $iEditcount;
+		}
+		return $aUsersEditsData;
+	}
+
+	private function mergeKey( $iUserId, $iWikiId ) {
+		return $iUserId . '@' . $iWikiId;
```

Comment:
Although you're not concatenating strings but integers :) Anyway, it's your call, I won't mind leaving it.